### DISCUSSION TITLE
How to write postBuildCommands?

### DISCUSSION THREAD

**Discussion by kivancbilen** [serverless/serverless#10185] [Q&A]
How can i write postBuildCommands in serverless.ts file, in serverless.yml it is possible, but i couldn't find any example regarding ts file.

**Comment by pgrzesik** [comment]
When using `serverless.ts` config, you need to export an Object or a promise that resolves to such object - it needs to have the same structure as in yaml so you can structure your object the same way. 

e.g. 

```
module.exports = {
  'provider': {
    'name': 'aws',
    'runtime': 'nodejs12.x',
  }
}
```

is the equivalent of

```
provider:
  name: aws
  runtime: nodejs12.x
```
